charts/controller/templates/controller-ingress-rule-controller.yaml

```diff
@@ -0,0 +1,21 @@
+{{ if .Values.global.experimental_native_ingress }}
```

Comment:
@krancour So I think your assumption isn't entirely accurate. I was able to test this under the following situation:

1. Define `global. experimental_native_ingress` in the `values.yaml` for **workflow**
2. Neglect to define `global.experimental_native_ingress` in the `values.yaml` for **controller** 
3. Nest the two charts together on my local filesystem as they are hosted https://charts.deis.com/workflow
```
kris-nova:workflow kris$ tree
.
├── Chart.yaml
├── charts
│   ├── builder
│   │   ├── Chart.yaml
│   │   ├── templates
│   │   │   ├── builder-controller-secret-key-auth.yaml
│   │   │   ├── builder-deployment.yaml
│   │   │   ├── builder-secret-ssh-private-keys.yaml
│   │   │   ├── builder-service-account.yaml
│   │   │   └── builder-service.yaml
│   │   └── values.yaml
│   ├── controller
│   │   ├── Chart.yaml
│   │   ├── templates
│   │   │   ├── controller-deployment.yaml
│   │   │   ├── controller-ingress-rules.yaml
│   │   │   ├── controller-secret-django-secret-key.yaml
│   │   │   ├── controller-service-account.yaml
│   │   │   ├── controller-service.yaml
│   │   │   └── deploy-hook-secret.yaml
│   │   └── values.yaml
```

And run a helm install locally 

```
helm install . --namespace deis --set global.experimental_native_ingress=true,controller.platform_domain=fabulous.af
```

And a quick check of our pods will demonstrate that the flag **in fact was inherited from the parent** (Notice the lack of a router)

```
kris-nova:workflow kris$ kubectl get po -n deis
NAME                                     READY     STATUS    RESTARTS   AGE
deis-builder-542482751-h844h             0/1       Running   1          1m
deis-controller-2957360810-sj2ik         0/1       Running   0          1m
deis-database-3140037217-i7fm5           1/1       Running   0          1m
deis-logger-176328999-8k11f              1/1       Running   2          1m
deis-logger-fluentd-620pz                1/1       Running   0          1m
deis-logger-fluentd-jx2fi                1/1       Running   0          1m
deis-logger-fluentd-ormq9                1/1       Running   0          1m
deis-logger-redis-2637728276-5e5ul       1/1       Running   0          1m
deis-minio-1095953168-w30z5              1/1       Running   0          1m
deis-monitor-grafana-432496062-rz5n6     1/1       Running   0          1m
deis-monitor-influxdb-2729657543-l63s4   1/1       Running   0          1m
deis-monitor-telegraf-if94d              1/1       Running   0          1m
deis-monitor-telegraf-mv4ul              1/1       Running   0          1m
deis-monitor-telegraf-pwcid              1/1       Running   0          1m
deis-nsqd-3597503299-flbfk               1/1       Running   0          1m
deis-registry-1455273266-5293m           1/1       Running   3          1m
deis-registry-proxy-hkpp5                1/1       Running   0          1m
deis-registry-proxy-hsouy                1/1       Running   0          1m
deis-registry-proxy-s9aaa                1/1       Running   0          1m
deis-workflow-manager-2528409207-z7xoh   1/1       Running   0          1m
```


---

I think we **should** add the directive to **both** charts (Which is already in the latest commit), so that they can be installed independently of each other.. but even after trying a second test with the directive defined in **both** charts, I was able to demonstrate that the value was passed down, and the parent chart's value took precedence over the child chart.

---

> I'm almost certain that the "parent" chart's global configuration isn't visible to this chart

A quick demonstration was able to debunk this assumption, hence why I always prefer to test things before making assumptions. 

##### Helm Version

```
kris-nova:workflow kris$ helm version
Client: &version.Version{SemVer:"v2.1.3", GitCommit:"5cbc48fb305ca4bf68c26eb8d2a7eb363227e973", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.1.3", GitCommit:"5cbc48fb305ca4bf68c26eb8d2a7eb363227e973", GitTreeState:"clean"}
```